libs/frontend/feature-build/src/components/BuildContent.vue

```diff
@@ -58,10 +63,25 @@ const innerStyle = computed<StyleValue>(() => {
 
   const innerHeight = buildContentWindowHeight / builderScale
 
+  // Set builder background/color to page root to handle the case
+  //   when child components overflow the root
+  let rootStyles: IRawStyle | undefined = undefined
+  if (activePage.value) {
```

Comment:
@abemscac This is the workaround for the editor. It's probably not very efficient to do it this way, but it seems to work for all cases.